### ISSUE TITLE
backend api tests fail with invalid postgres password

### ISSUE THREAD

**Issue by martinjrobins** [Battery-Intelligence-Lab/galvanalyser#49]
**Describe the bug**

running the backend api tests fails with this error:

```
django.db.utils.OperationalError: connection to server at "postgres" (10.89.1.86), port 5433 failed: FATAL:  password authentication failed for user "postgres"
```

**Comment by martinjrobins** [comment]
Once I set the postgres password in `docker-compose.test.yml` to match what I had in `.env.secret` this worked ok

**Comment by mjaquiery** [comment]
@martinjrobins, Please confirm this was closed by #50

**Comment by martinjrobins** [comment]
fixed, thanks @mjaquiery
